lib/main.dart

```diff
@@ -28,10 +29,13 @@ class MyApp extends StatelessWidget {
           canvasColor: canvasTheme,
           fontFamily: 'Lato',
         ),
-        home: ProductsOverviewScreen(),
+        initialRoute: ProductsOverviewScreen.routeName,
+        onGenerateRoute: (settings) => MaterialPageRoute<dynamic>(builder: (ctx) => ProductsOverviewScreen()),
+        onUnknownRoute: (settings) => MaterialPageRoute<dynamic>(builder: (ctx) => ProductsOverviewScreen()),
         routes: {
           ProductDetailScreen.routeName: (ctx) => ProductDetailScreen(),
           CartScreen.routeName: (ctx) => CartScreen(),
+          OrdersScreen.routeName: (ctx) => OrdersScreen(),
         },
```

Comment:
this can be improved with the approach of creating a separate class that generates routes according to its route name.

see this comment: https://github.com/MapleSyyrup/MyShop-App/pull/10/files#r551092174

P.S.: Please be mindful of the scope of comment (Comment on lines this to that)